lang/attribute/event/src/lib.rs

```diff
@@ -47,13 +42,13 @@ pub fn event(
     };
 
     #[cfg(feature = "idl-build")]
-    {
+    let ret = {
```

Comment:
Not very important but this pattern is being used in other places in the repo so it would be better if style changes here are reverted in order to keep it consistent.